build.gradle

```diff
@@ -38,6 +38,7 @@ if (clang_versions.length == 0) {
 }
 def clang_version = clang_versions[0]
 
+def buildDir = layout.buildDirectory.get()
```

Comment:
`$buildDir` is deprecated in the latest version of Gradle. This is a workaround.